drivers/include/net/netdev2.h

```diff
@@ -74,6 +74,9 @@ typedef enum {
     NETDEV2_EVENT_RX_COMPLETE,      /**< finished receiving a packet */
     NETDEV2_EVENT_TX_STARTED,       /**< started to transfer a packet */
     NETDEV2_EVENT_TX_COMPLETE,      /**< finished transferring packet */
+#ifdef MODULE_OPENTHREAD
+    NETDEV2_EVENT_TX_COMPLETE_DATA_PENDING, /**< finished transferring packet and has data pending flag **/
```

Comment:
@miri64 Not sure what you mean. I introduced `TX_NOACK` and `TX_MEDIUM_BUSY` in #3729.